trino/client.py

```diff
@@ -455,6 +455,7 @@ def http_headers(self) -> Dict[str, str]:
         headers[constants.HEADER_USER] = self._client_session.user
         headers[constants.HEADER_TIMEZONE] = self._client_session.timezone
         headers[constants.HEADER_CLIENT_CAPABILITIES] = 'PARAMETRIC_DATETIME'
+        headers["User-Agent"] = f"Trino Python Client/{__version__}"
```

Comment:
is there a constant that request provides for the header name?

It may not matter if requests normalises the keys of the dict when merging the headers.

Reply:
Also it seems we can test this via the mocked client in `test_client.py`?

Reply:
There is no constant for that in `requests` (also see [examples](https://requests.readthedocs.io/en/latest/user/quickstart/#custom-headers)) but I think it's more readable to have it directly in dictionary. I can create a constant for the text `Trino Python Client`, shall I?

I will add the test 👌

Reply:
tests are added.